services/docker-protonet-network.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Docker protonet bridge network initialization
+After=docker.service
+
+[Service]
+ExecStart=-/usr/bin/env docker network create --driver bridge protonet
+ExecStartPost=/bin/bash -c "docker network inspect protonet | jq '.[0].IPAM.Config[0].Gateway' --raw-output | cut -d '/' -f1 > /etc/protonet/system/dockerhost"
+KillMode=none
+Type=oneshot
```

Comment:
No, it isn't. I'll have to hunt down where I took this from, I took one of the existing oneshot commands as a template